doc/source/rllib/env-to-module-connector.rst

```diff
@@ -0,0 +1,546 @@
+.. include:: /_includes/rllib/we_are_hiring.rst
+
+.. include:: /_includes/rllib/new_api_stack.rst
+
+
+.. _env-to-module-pipeline-docs:
+
+Env-to-module pipelines
+=======================
+
+One env-to-module pipeline resides on each :py:class:`~ray.rllib.env.env_runner.EnvRunner` and is responsible
+for handling the data flow from the `gymnasium.Env <https://gymnasium.farama.org/api/env/>`__ to
+the :py:class:`~ray.rllib.core.rl_module.rl_module.RLModule`.
+
+.. figure:: images/connector_v2/env_runner_connector_pipelines.svg
+    :width: 1000
+    :align: left
+
+    **EnvRunner ConnectorV2 Pipelines**: Both env-to-module and module-to-env pipelines are located on the :py:class:`~ray.rllib.env.env_runner.EnvRunner`
+    workers. The env-to-module pipeline sits between the RL environment, a `gymnasium.Env <https://gymnasium.farama.org/api/env/>`__, and the
+    :py:class:`~ray.rllib.core.rl_module.rl_module.RLModule`, and translates ongoing episodes into batches for the model's `forward_...()` methods.
+    The module-to-env pipeline serves the other direction, converting the output of the :py:class:`~ray.rllib.core.rl_module.rl_module.RLModule`, such as
+    action logits and action distribution parameters, to actual actions understandable by the `gymnasium.Env <https://gymnasium.farama.org/api/env/>`__
+    and used in the env's next `step()` call.
+
+The env-to-module pipeline, when called, performs translations from a list of ongoing :ref:`Episode objects <single-agent-episode-docs>` to an
```

Comment:
Imo "transformations" would suit here better b/c translation is either a special geometric transformation of values/vectors or is rather used for changes in the system/language of an object (like dates)